web/src/views/PublicDandisetsView/PublicDandisetsView.vue

```diff
@@ -1,11 +1,96 @@
 <template>
-  <v-container>
-    TODO: PublicDandisetsView
-  </v-container>
+  <div>
+    <v-toolbar
+      color="grey darken-2 white--text"
+    >
+      <v-toolbar-title class="d-none d-md-block mx-8">
+        Public Dandisets
+      </v-toolbar-title>
+      <v-divider
+        class="d-none d-md-block"
+        vertical
+      />
+      <div class="mx-6">
+        Sort By:
+      </div>
+      <v-chip-group
+        v-model="selection"
+        active-class="white light-blue--text"
+        dark
+        mandatory
+      >
+        <v-chip
+          v-for="option in options"
+          :key="option.name"
+          @click="setSort(option.sort)"
+        >
+          {{ option.name }}
+        </v-chip>
+      </v-chip-group>
+      <SearchField />
+    </v-toolbar>
+    <DandisetList
+      class="
+        mx-12
+        my-12"
+      :dandisets="$store.state.publicDandisets.dandisets"
+    />
+    <v-pagination
+      v-model="$store.state.publicDandisets.page"
+      :length="$store.state.publicDandisets.pages"
+      :value="$store.state.publicDandisets.page"
```

Comment:
The `v-model` on line 39 will directly modify state, which is a strict anti-pattern when it comes to vuex. If you need to model something to the state, I'd look at [computed setters](https://vuejs.org/v2/guide/computed.html#Computed-Setter). That way, you can commit a mutation if you need to change the value. If doing that, you can't mix it with `mapState`, so you'd just have to directly reference `$store.state.publicDandisets.page` in the getter.

In general, I try to avoid referencing `$store` directly in the template if I can. Even if you use a computed getter/setter for `page`, I still think it'd be worth using `mapState` on `dandisets` and `pages`, and referencing those mapped props here.

I'm curious what @brianhelba's opinion on this is though.

Reply:
I fixed this, I don't know what I was thinking.

I think that is an excellent policy.

Reply:
I also try to use (and I'm typically able to) the `map` helpers everywhere, and avoid `$store.state` entirely. It helps make it much more clear (to me at least) which parts of my store surface a given component needs.